flexmeasures/data/models/planning/shiftable_load.py

```diff
@@ -0,0 +1,217 @@
+from __future__ import annotations
+
+from math import ceil
+from datetime import timedelta
+import pytz
+
+import pandas as pd
+
+from flexmeasures.data.models.planning import Scheduler
+
+from flexmeasures.data.queries.utils import simplify_index
+from flexmeasures.data.models.time_series import Sensor
+from flexmeasures.data.schemas.scheduling.shiftable_load import (
+    ShiftableLoadFlexModelSchema,
+    LoadType,
+    OptimizationSense,
+)
+from flexmeasures.data.schemas.scheduling import FlexContextSchema
+
+
+class ShiftableLoadScheduler(Scheduler):
+
+    __version__ = "1"
+    __author__ = "Seita"
+
+    def compute(self) -> pd.Series | None:
+        """Schedule a load, defined as a `power` and a `duration`, within the specified time window.
+        For example, this scheduler can plan the start of a process that lasts 5h and requires a power of 10kW.
+
+        This scheduler supports three types of `load_types`:
+            - Inflexible: this load requires to be scheduled as soon as possible.
+            - Breakable: this load can be divisible in smaller consumption periods.
+            - Shiftable: this load can start at any time within the specified time window.
+
+        The resulting schedule provides the power flow at each time period.
+
+        Parameters
+        ==========
+
+        cost_sensor: it defines the utility (economic, environmental, ) in each
```

Comment:
I was looking for this description in the flex model. Well, in the end, we need it in the documentation page for this scheduler (currently API notation section, soon dedicated scheduling section).

Reply:
Yes, good point. I've added it to the notation section.